packages/com.circleci.v2/Config.pkl

```diff
@@ -813,6 +813,8 @@ output {
   renderer = new YamlRenderer {
     converters {
       [AbstractStep] = (it) -> Map(it.__name__, it.toMap())
+      [Duration] = (it) ->
+        let (unit = if (it.unit == "min") "m" else it.unit) "\(it.value)\(unit)"
```

Comment:
Hm, not sure if CircleCI accepts days, nanoseconds, microseconds or milliseconds. Let's coerce these to units that are described by the documentation.

```suggestion
        let (dur =
          if (it.unit is "d") it.toUnit("min")
          else if (it.unit is "us"|"ms"|"ns") it.toUnit("s")
          else it
        )
        let (unit = if (dur.unit == "min") "m" else dur.unit)
          "\(dur.value)\(unit)"
```